### PR TITLE
MC-772 documenting prometheus basic auth

### PR DESCRIPTION
**Pull request by erosb**
related MC PR: https://github.com/hazelcast/management-center/pull/4515

### CODE REVIEW

File: docs/modules/ROOT/pages/prometheus.adoc [hunk 1]
```diff
@@ -64,3 +64,24 @@ scrape_configs:
...
+    - targets: ['localhost:8080']
+    basic_auth:
+      username: my-user
+      password: my-pass
+
+----
+
+
+
+You can reset the authentication with the `mc-conf.sh security reset` command, which will also delete all users from the database.
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/prometheus.adoc:87]: Prometheus supports basic authentication to secure the scrape endpoint.
First, you need to set it up using the following MC-Conf command

```
mc-conf.sh set prometheus-auth <my-user> <my-pass>
```

Then, you need to configure Prometheus as follows:

[source,yaml]
----
  - job_name: 'HZ MC'
    static_configs:
    - targets: ['localhost:8080']
    basic_auth:
      username: my-user
      password: my-pass
----

You can reset the authentication with the `mc-conf.sh security reset` command, which also deletes all the users from the database.

Comment (Serdaro) [reply docs/modules/ROOT/pages/prometheus.adoc:87]: Let's restructure it as above. Minor wondering; are the my-user and my-pass the credentials to login to MC or they create a new user?

**Review by Serdaro — COMMENTED** [review]
Minor restructure.

**Comment by erosb** [comment]
> Let's restructure it as above.

Thanks, done.

> Minor wondering; are the my-user and my-pass the credentials to login to MC or they create a new user?

No, these credentials are completely separate from the user authentication.

**Review by Serdaro — APPROVED** [review]

**Pull request merged**